src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContext.cs

```diff
@@ -17,20 +18,26 @@ internal sealed class JobHostContext : IDisposable
         private readonly IListener _listener;
         private readonly TraceWriter _trace;
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _fastLogger; // optional
+        private readonly IAsyncCollector<FunctionResultLog> _aggregator;
```

Comment:
It's temporary. We can't just delete support for loggers -- next major version update we'll pull what we can.